docs: add Wordpress special handling info about wp-cli.yml (#7080)
ddev/ddev: docs/content/users/usage/cms-settings.md

```

See also [TYPO3 Documentation](https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/SiteHandling/BaseVariants.html).

### WordPress Specifics

> #### WP-CLI and changing webroot in config.yaml

Reviewer: DDEV uses `docroot` for this.

```suggestion
#### WP-CLI and changing the docroot in config.yaml
```

> If you use something other than the root directory (`''`) for your webroot, wp-cli will not work properly. To fix this, create a `wp-cli.yml` file in the root directory that contains `path: {webroot/path}`.

Reviewer: ```suggestion
If you use something other than the root directory (`''`) for your docroot, wp-cli will not work properly. To fix this, create a `wp-cli.yml` file in the root directory that contains `path: {<docroot>/path}`.
```

Reviewer: Don't understand why you marked it resolved without changing the usage from webroot to docroot.

> For example, if your webroot is `public`, your `wp-cli.yml` file should contain:

Reviewer: ```suggestion
For example, if your docroot is `public`, your `wp-cli.yml` file should contain:
```
